tasks/configureApache.yml

```diff
@@ -75,12 +61,12 @@
 
   - name: Copy key for host
     copy:
-      remote_src: "{{ ( use_native_cert ) | ternary( 'True','False' ) }}"
+      remote_src: "{{ ( use_native_cert ) | ternary( 'true','false' ) }}"
```

Comment:
Why do you need to substitute one boolean value with another?

Reply:
Fixed